tests/ImageStreamIO_subTest_Operations.cpp

```diff
@@ -342,6 +363,25 @@ ImageStreamIO_subTest_Operations(int& test_count, int& success_count)
     return;
 }
 
+/*
+ * Build standalone from source in this repo:
+ *
+ *   g++ -D__ISIOUT_OPS_TEST_MAIN__=main      \
+ *       -I..                                 \
+ *       ImageStreamIO_subTest_Operations.cpp \
+ *       ../ImageStreamIO.c                   \
+ *       -pthread                             \
+ k       -o ImageStreamIO_subTest_Operations
+ *
+ * Build standalone against ImageStreamIO shared library and header file
+ * under /usr/local/:
+ *
+ *   g++ -D__ISIOUT_OPS_TEST_MAIN__=main      \
+ *       -I/usr/local/include/ImageStreamIO   \
+ *       ImageStreamIO_subTest_Operations.cpp \
+ *       -lImageStreamIO -pthread             \
+ *       -o ImageStreamIO_subTest_Operations
+ */
```

Comment:
This module, ImageStreamIO_subTest_Operation.cpp, defaults to being called by a Gtest, but it can be also compiled as a standalone program; these comments provide the commands to do the latter.